lib/CLI/Parser.hs

```diff
@@ -318,9 +309,9 @@ tagOptions =
 -- Common
 ----------------------------------------------------------------------------
 
-readPath' :: Text -> Either String Path
+readPath' :: Text -> Either Text Path
```

Comment:
We should do the same to `readEntryPath'` for consistency, change `Either String` to `Either Text`

Reply:
Done